GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -803,7 +802,7 @@ private void diffUpdateSubmoduleMenuItem_Click(object sender, EventArgs e)
 
             foreach (var item in unStagedFiles.Where(it => it.IsSubmodule))
             {
-                //TBD FormProcess.ShowDialog(this, GitCommandHelpers.SubmoduleUpdateCmd(item.Name));
+                FormProcess.ShowDialog(null, Module, GitCommandHelpers.SubmoduleUpdateCmd(item.Name));
```

Comment:
Shouldn't the first argument set to `this`?

Reply:
No, that was the problem as FormBrowse is derived from GitModuleForm and RevisionDiff is GitModuleControl and there is no ShowDialog override.
The null in this override is IWin32Window, OK to set to null.

Reply:
Should it be `this.FindForm()` then?

Reply:
Maybe that will work too but I have not got the hang of how it could be used.

I used a syntax that is used in other situations. There is no need of getting the results here so this should be sufficient.

Reply:
The following works, updating

FormProcess.ShowDialog((FindForm() as FormBrowse), GitCommandHelpers.SubmoduleUpdateCmd(item.Name));